README.md

```diff
@@ -1 +1,129 @@
+[![Build Status](https://travis-ci.com/oasis-roles/vmware_provision.svg?branch=master)](https://travis-ci.com/oasis-roles/vmware_provision)
 
+vmware_provision
+===========
+
+Provisions VMWare hosts and adds them to inventory groups
+
+Requirements
+------------
+
+Ansible 2.8 or higher
+
+Red Hat Enterprise Linux 7 or equivalent
+
+Valid Red Hat Subscriptions
+
+Python pip package `pyvmomi`
+
+Role Variables
+--------------
+
+Currently the following variables are supported:
+
+### General
+
+* `vmware_provision_become` - Default: true. If this role needs administrator
+  privileges, then use the Ansible become functionality (based off sudo).
+* `vmware_provision_become_user` - Default: root. If the role uses the become
+  functionality for privilege escalation, then this is the name of the target
+  user to change to.
+* `vmware_provision_hostname` - The hostname or IP address of the vSphere vCenter or ESXi server. **REQUIRED**
+* `vmware_provision_username` - The username of the vSphere vCenter or ESXi server. **REQUIRED**
+* `vmware_provision_password` - The password of the vSphere vCenter or ESXi server. **REQUIRED**
+* `vmware_provision_custom_VMs` - A list of custom VM configurations to deploy to a vSphere vCenter or ESXi server. Each list item represents a VM to be provisioned. Parameters of each list item should have key/value pairs that match the arguments to the [vmware_guest](https://docs.ansible.com/ansible/latest/modules/vmware_guest_module.html#vmware-guest-module) module for Ansible.
```

Comment:
This and the next variable have upper case letters. I don't believe we've used those before. If we have, then I'm fine with it here, but I don't recall it in the past.

But this line and line 35 are both epically long. You can introduce some basic newlines in here, please, for readability and editability. We don't have a hard line limit on the *.md files, but in general it's nice to have them wrap in a similar place to other text types around the 80-100 character range.